client/components/map/mapToolTip.jsx

```diff
@@ -6,9 +6,9 @@ const Tooltip = ({distance, trailName, isActive, ...props}) => (
       <h5>
         { trailName || 'No name given' }
       </h5>
-      <p>Estimated Distance: {props.isActive 
-      ? guessLength(props.popupContainer, distance.addLength) + 'Miles'
-      : '' 
+      <p>Estimated Distance: {isActive.b 
```

Comment:
This was this way to demonstrate everything loading at once, instead of on `isActive`. It was a quick hack.